### DISCUSSION TITLE
polaris team? contributer rank?

### DISCUSSION THREAD

**Discussion by spazestudios** [NovaoTech/solaris#22] [Ideas]
you know how theres a scratch team
you know how theres ranks on scratch example: scratchteam, scratcher, new scratcher etc
will there be a contributer rank? will there be a polaris team? will there be mods/admins? etc

**Comment by MikeDev101** [comment]
To be honest, this system "just works", so I don't think we need to radically change it. Although, of course, change some text for branding.

"New Member", "Member", and "Polaris Team".

**Comment by icarusinthewings** [comment]
I think we'll have a "Stargazer" rank for the new members rank, an "Astronomer" rank for members, and a "Rocket Scientist" rank for contributors.
